crpg.py

```diff
@@ -102,7 +106,11 @@ def generate(filename):
             if n_type == "fight" or n_type == "run":
                 args.append(game.player)
 
+            if n_type == "node":
+                args.append(game.player)
+
```

Comment:
Nodes should not affect the player. Actions, however, are built explicitly to do just that!